smart_contract_examples/SwapExamples.sol

```diff
@@ -139,7 +139,7 @@ contract SwapExamples {
         TransferHelper.safeApprove(DAI, address(swapRouter), amountInMaximum);
 
         // The parameter path is encoded as (tokenOut, fee, tokenIn/tokenOut, fee, tokenIn)
-        // The tokenIn/tokenOut field is the shared token between the two pools used in the multiple pool swap. In this case USDC is the "shared" token.
+        // The tokenIn/tokenOut field is the shared token between the two pools used in the multiple pool swap. In this case, USDC is the "shared" token.
         // For an exactOutput swap, the first swap that occurs is the swap which returns the eventual desired token.
```

Comment:
How about using `eventually` instead of `eventual`?

Reply:
"For an exactOutput swap, the first swap that occurs is the swap that returns the eventually desired token."